meetings/2025-01-21-taskforce-meeting.md

```diff
@@ -0,0 +1,135 @@
+# 2025-01-21: ActivityPub Trust and Safety Task Force Meeting
+
+## Attending
+
+- Emelia Smith (thisismissem@hachyderm.io)
+- Dmitri Z.
+- Sebastian Lasse [@sl@digitalcourage.social](https://digitalcourage.social/@sl007) (he/they)
+- bumblefudge (first time caller)
+- Jaz-Michael King (https://mastodon.iftas.org/@jaz)
+- Robert W Gehl (he/him) (@rwg@aoir.social)
+
+## Agenda
+
+1. IP Protection Note Reminder:
+    - Anyone can participate in these calls. However, all substantive contributors to any Taskforce Work Items must be members of the Social Web CG with full IPR agreements signed. https://www.w3.org/community/socialcg/join
+    - To contribute to Work Items: ensure you have a W3 account: https://www.w3.org/accounts/request, and sign the W3C Community Contributor License Agreement (CLA): https://www.w3.org/community/about/agreements/cla/
+2. Reminder of [Code of Conduct](https://github.com/swicg/activitypub-trust-and-safety/blob/main/CODE_OF_CONDUCT.md)
+3. Introductions, if necessary
+4. FOSDEM Announcement
+5. Review [#51](https://github.com/swicg/activitypub-trust-and-safety/pull/51)
+6. Discuss [#4](https://github.com/swicg/activitypub-trust-and-safety/issues/4), particularly annotations (possibly, if there isn't more pressing concerns)
+
+## Minutes
+
+- Emelia: Introduced the meeting
+
+- Emelia: First item on our list of things to discuss is pull request 51. It is a draft of what I think the initial report structure could be. 
+
+- Emelia: Notes that the pull request can be previewed, and that previews only work for the initial report document at the moment.
+
+- Emelia: So yeah, I'd like to go through that and seek feedback. If there's any comments, concerns, questions? 
+
+- Emelia: I've written some of the text that I think should be in the initial report, obviously, it's an early draft. It's more the structure and what we want to make. I do think that the introduction is more or less the direction.
+
+- Robert: I thought the structure looks very good. Seems to capture a lot of the issues that I've seen this group talk about. I imagine in the writing things could grow, but at least it's a good skeleton for me.
+
+- Emelia: Yeah, one section that I did want to add, which is kind of inspired by the... recent post by Mastodon.art about Pixelfed is an absolute bare minimum requirements section. 
```

Comment:
is there context on this?

Reply:
https://mastodon.art/@Curator/113866607769121751

I don't know if I should include the URL in the notes?

Reply:
Maybe briefly summarize the content of the post? "inspired by a recent post about Mastodon.art about Pixelfed [having some bugs in their moderation workflow]" maybe? That way it's more clear what you're referencing if someone reads these meeting notes some months or years from now.

Reply:
```suggestion
- Emelia: Yeah, one section that I did want to add, which is kind of inspired by the [recent post by Mastodon.art about Pixelfed](https://mastodon.art/@Curator/113866607769121751) is an absolute bare minimum requirements section. 
```

Reply:
That sounds fair. Wasn't sure if I should really link it, hence not originally.